model-optimizer/extensions/front/GeLUMerger_Tanh_test.py

```diff
@@ -15,11 +15,10 @@
 """
 
 import unittest
+from math import sqrt
```

Comment:
Please, update copyright year in this file.

Reply:
Fixed